src/app/components/initial-setup-nav/initial-setup-nav.component.ts

```diff
@@ -19,13 +19,12 @@ export class InitialSetupNavComponent implements OnInit {
   }
 
   handleButtonClick(): void {
-    if (this.nextPage === '/initial-setup-tips') {
+    if (this.nextPage === '/setup/tips') {
       this.dataservice.setSetupCompleted(true);
     }
     if (this.username) {
-    const input = document.getElementById('inputName');
-    const text = input.innerHTML;
-    this.dataservice.setUsername(text);
+    const input = (document.getElementById('inputUsername') as HTMLInputElement).value;
+    this.dataservice.setUsername(input);
     }
   }
```

Comment:
one last thing: I feel like the coloring of the init-nav dots should be inverted, the active state should be lighter and the to come should be dark, so when you are on the second page:

```
light - light - dark
```

Reply:
done